packages/components/src/shared/src/useFocusManager.ts

```diff
@@ -141,18 +145,20 @@ export class FocusManager {
     focusNext({ canFocus, ...options }: FocusOptions = {}) {
         const { elements } = this.scope;
 
+        const visibleElements = this.getVisibileElements(elements);
```

Comment:
Au lieu de filtré les elements localement, on pourrait plutot essayer de fixer le issue à la source. 

L'inspiration de ce concept viens de spectrum. Ils semble ajouter des selectors qu'on a pas, par exemple 

const TABBABLE_ELEMENT_SELECTOR = focusableElements.join(':not([hidden]):not([tabindex="-1"]),');

https://github.com/adobe/react-spectrum/blob/34717616f3a6555d3f4d6739635d3ac347b50a58/packages/%40react-aria/focus/src/FocusScope.tsx#L224


tu peux regarder dans focusableTreeWalker.ts ln 21. Tu vas voir que c'est pas mal semblable à ce que je t'ai envoyé

Reply:
I agree with Alex.